tensorbay/client/version.py

```diff
@@ -643,13 +643,13 @@ def _delete_job(self, job_id: str) -> None:
 
         """
 
-    def _generate_jobs(
+    def _list_jobs(
```

Comment:
Why the `_generate_jobs` changes to `_list_jobs`?

Reply:
Here we do not need to get a generator. We will add the _generate job to SquashAndMerge.
SquashAndMerge.list_jobs -> SquashAndMerge._generate_jobs -> JobMixin._list_jobs